localstack-core/localstack/services/apigateway/next_gen/execute_api/handlers/integration_request.py

```diff
@@ -105,20 +114,20 @@ def render_request_template_mapping(
         self,
         context: RestApiInvocationContext,
         template: str,
-    ) -> tuple[str, ContextVarsRequestOverride]:
+    ) -> tuple[bytes, ContextVarsRequestOverride]:
         request: InvocationRequest = context.invocation_request
-        body = request.get("body", b"").decode()
+        body = request.get("body")
```

Comment:
was the removal of the fallback value on purpose? I guess `body` maybe can't really empty? should we access it directly it without `.get()` to be sure we never have `None`?

Reply:
I guess it can't be, I wasn't totally sure but I guess it can't

Reply:
The fallback was originally to protect the `decode` but since using `to_str` the fallback was no longer required

Reply:
Ah, I see! I wasn't sure if the VTL template could take `None` for the `input.body` or not, but I guess it won't be none and just `b""`!